frontend/src/views/MyProgress/index.vue

```diff
@@ -1,88 +1,117 @@
 <template>
   <div class="text-left">
     <BreadcrumbsNav :links="[{ to: { name: 'DashboardManager' }, title: canteen ? canteen.name : 'Dashboard' }]" />
-    <ProductionTypeTag v-if="canteen" :canteen="canteen" class="mt-n2" />
-    <h1 class="fr-h3 my-2" v-if="canteen">{{ canteen.name }}</h1>
-    <v-row v-if="canteenPreviews.length > 1">
-      <v-col>
-        <v-btn outlined color="primary" class="fr-btn--tertiary" :to="{ name: 'ManagementPage' }">
-          Changer d'établissement
-        </v-btn>
+    <v-row>
+      <v-col cols="12" md="9">
+        <ProductionTypeTag v-if="canteen" :canteen="canteen" class="mt-n2" />
+        <!-- TODO: put Ma progression text up here -->
+        <h1 class="fr-h3 my-2" v-if="canteen">{{ canteen.name }}</h1>
+        <v-row v-if="canteenPreviews.length > 1">
+          <v-col>
+            <v-btn outlined color="primary" class="fr-btn--tertiary" :to="{ name: 'ManagementPage' }">
+              Changer d'établissement
+            </v-btn>
+          </v-col>
+        </v-row>
       </v-col>
-    </v-row>
-    <v-row v-if="canteen" class="mt-10">
-      <v-col cols="12" sm="3" md="2" style="border-right: 1px solid #DDD;">
-        <h2 class="fr-h5">Ma progression</h2>
-        <nav aria-label="Année du diagnostic" v-if="canteen && $vuetify.breakpoint.smAndUp">
-          <v-list nav class="text-left">
-            <v-list-item-group>
-              <v-list-item
-                :ripple="false"
-                :to="{ name: 'MyProgress', params: { year } }"
-                v-for="year in years"
-                :key="year"
-              >
-                <v-list-item-title>
-                  {{ year }}
-                  <span v-if="year >= currentYear" class="fr-text-xs font-weight-normal">(prévisionnel)</span>
-                </v-list-item-title>
-              </v-list-item>
-            </v-list-item-group>
-          </v-list>
-        </nav>
-        <DsfrSelect hide-details label="Année" :items="years" v-model="year" v-else-if="canteen" />
+      <v-col cols="12" sm="5" md="3">
+        <!-- TODO: should this be a nav element? -->
```

Comment:
[With appropriate CSS, this can be presented as a sidebar, navigation bar, or drop-down menu.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)

The benefit being that

> User agents, such as screen readers targeting disabled users, can use this element to determine whether to omit the initial rendering of navigation-only content.

We used a nav element for the old sidebar. But now that they aren't actually links, although changing it does change the URL, I'm not sure if that would break things. MDN does say it accepts all flow content though.

I'll just remove the comment for now, maybe one day I'll learn to write semantic HTML 😞